runtime/moonbase/src/lib.rs

```diff
@@ -599,7 +599,7 @@ impl pallet_crowdloan_rewards::Config for Runtime {
 }
 
 parameter_types! {
-	pub const DepositAmount: Balance = 100;
+	pub const DepositAmount: Balance = 100 * currency::UNITS;
```

Comment:
Good catch.
We also need to tell the partners they need to keep those token for registration